### PR TITLE
Feat: Formatter

### PR DESCRIPTION
**Pull request by Meldiron**
Add formatter, and automated tests.

### CODE REVIEW

File: runtimes/node/versions/latest/src/server.js [hunk 1]
```diff
@@ -2,233 +2,289 @@ const micro = require("micro");
...
+	);
 
-    try {
-        await action(logger, req, res);
-    } catch (e) {
-        logger.write(e, Logger.TYPE_ERROR);
+	try {
+		await action(logger, req, res);
+	} catch (e) {
+		logger.write(e, Logger.TYPE_ERROR);
```
Comment (christyjacob4) [inline runtimes/node/versions/latest/src/server.js:16]: The default indentation for js seems to be a bit too much . Can we reduce it?

**Review by christyjacob4 — APPROVED** [review]
Looks good. I think we should check the default indentation for JS and reduce it. TS looks fine 👍

File: runtimes/node/versions/latest/src/server.js (hunk 1 above)
Comment (Meldiron) [reply runtimes/node/versions/latest/src/server.js:16]: It's same for Bun. It's 1 tab. I think it's okay, that's same as 4 spaces. And There isn't clear winner between 2 and 4 spaces.

Comment (Meldiron) [reply runtimes/node/versions/latest/src/server.js]: I guess GitHub renders tab badly. Switched to spaces in biome.js config file

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**